app/src/pages/patientResultPage/PatientResultPage.tsx

```diff
@@ -22,25 +22,28 @@ function PatientResultPage() {
     const { handleSubmit } = useForm();
 
     const submit = () => {
-        if (userIsGPAdmin || userIsGPClinical) {
+        if (userIsPCSE) {
+            // Make PDS and Dynamo document store search request to download documents from patient
+            navigate(routes.ARF_OVERVIEW);
+        } else {
             // Make PDS patient search request to upload documents to patient
             if (typeof patientDetails?.active === 'undefined') {
                 setInputError('We cannot determine the active state of this patient');
                 return;
             }
-            if (patientDetails?.active) {
-                navigate(routes.LLOYD_GEORGE);
-            } else if (userIsGPAdmin) {
-                navigate(routes.ARF_UPLOAD_DOCUMENTS);
+
+            if (patientDetails?.deceased) {
+                navigate(routeChildren.PATIENT_ACCESS_AUDIT_DECEASED);
             } else {
-                navigate(routes.SEARCH_PATIENT);
+                if (patientDetails?.active) {
+                    navigate(routes.LLOYD_GEORGE);
+                } else if (userIsGPAdmin) {
+                    navigate(routes.ARF_UPLOAD_DOCUMENTS);
+                } else {
+                    navigate(routes.SEARCH_PATIENT);
+                }
```

Comment:
Do we need this nested in an else?
```suggestion
                else if (patientDetails?.active) {
                    navigate(routes.LLOYD_GEORGE);
                } else if (userIsGPAdmin) {
                    navigate(routes.ARF_UPLOAD_DOCUMENTS);
                } else {
                    navigate(routes.SEARCH_PATIENT);
                }
```

Reply:
adjusted